editor/src/clj/editor/model.clj

```diff
@@ -529,15 +529,16 @@
   (when (migrated? model-node-id model-desc evaluation-context)
     (g/flag-nodes-as-migrated! evaluation-context [model-node-id])))
 
-(defn load-model [_project self resource {:keys [name default-animation mesh skeleton animations materials create-go-bones] :as model-desc}]
+(defn load-model [_project self resource {:keys [materials] :as model-desc}]
   (concat
-    (g/set-properties self
-      :name name
-      :default-animation default-animation
-      :mesh (workspace/resolve-resource resource mesh)
-      :skeleton (workspace/resolve-resource resource skeleton)
-      :animations (workspace/resolve-resource resource animations)
-      :create-go-bones create-go-bones)
+    (let [resolve-resource #(workspace/resolve-resource resource %)]
+      (gu/set-properties-from-pb-map self ModelProto$ModelDesc model-desc
+        name :name
+        default-animation :default-animation
+        mesh (resolve-resource :mesh)
+        skeleton (resolve-resource :skeleton)
+        animations (resolve-resource :animations)
+        create-go-bones :create-go-bones))
```

Comment:
`create-go-bones` could be nil, and that would cause the value to be set to nil, and then it was displayed as an indeterminate check-box in the editor.